cmd/swapcli/main.go

```diff
@@ -1096,7 +1096,10 @@ func runGetVersions(ctx *cli.Context) error {
 	fmt.Printf("swapd: %s\n", resp.SwapdVersion)
 	fmt.Printf("p2p version: %s\n", resp.P2PVersion)
 	fmt.Printf("env: %s\n", resp.Env)
-	fmt.Printf("swap creator address: %s\n", resp.SwapCreatorAddr)
+	// Bootnodes don't have a contract address
+	if resp.SwapCreatorAddr != nil {
+		fmt.Printf("swap creator address: %s\n", resp.SwapCreatorAddr)
+	}
```

Comment:
Before this change it was printing `swap creator address: <nil>` for bootnodes. Now it just won't print that line for bootnodes.